src/Apps/Order/routes.tsx

```diff
@@ -80,7 +59,7 @@ export const routes: RouteConfig[] = [
     children: [
       {
         path: "respond",
-        Component: RespondRoute,
+        getComponent: () => loadable(() => import("./Routes/Respond")),
```

Comment:
I'm sure this is fine, but can you explain what this change is for?

Reply:
This is bundle splitting. When webpqck encounters a dynamic `import()` keyword it will generate a new .js file, that contains all the sources for that route. That way, if a user never encounters this path, they wont need to download the JS. We're now splitting at all of our routes. The `loadable` aspect is is a wrapper around [loadable-components](https://github.com/gregberge/loadable-components), it provides some additional features but most importantly helps us manage SSR.